lib/constants.js

```diff
@@ -125,7 +125,7 @@ const InternalContractsIDs = {
   // CL: 9, // CoverMigrator.sol - removed
   AS: 10, // Assessment.sol
   CI: 11, // IndividualClaims.sol - Claims for Individuals
-  CG: 12, // YieldTokenIncidents.sol - Claims for Groups
+  // CG: 12, // YieldTokenIncidents.sol - Claims for Groups
```

Comment:
just rename

Reply:
Here it will not be consistent with previous removals which are commented out?

Reply:
lets keep it as commented out to be consistent